### PR TITLE
[kafka output] Rand integer partitioner

### PR DESCRIPTION
**Pull request by nezdolik**

### CODE REVIEW

File: modules/kafka/src/main/java/com/spotify/ffwd/kafka/KafkaPartitioner.java [hunk 1]
```diff
@@ -22,6 +22,7 @@
 import com.spotify.ffwd.model.Event;
 import com.spotify.ffwd.model.Metric;
 import java.util.Optional;
+import java.util.Random;
 import java.util.function.Supplier;
 import java.util.zip.CRC32;
```
Comment (udoprog) [inline modules/kafka/src/main/java/com/spotify/ffwd/kafka/KafkaPartitioner.java]: Must add it here for it to work in configuration.

File: modules/kafka/src/main/java/com/spotify/ffwd/kafka/KafkaPartitioner.java [hunk 2]
```diff
@@ -115,4 +116,28 @@ public int partition(final Metric metric, final String defaultHost) {
             return Hashed::new;
         }
     }
+
+    class RandInt implements KafkaPartitioner {
```
Comment (udoprog) [inline modules/kafka/src/main/java/com/spotify/ffwd/kafka/KafkaPartitioner.java]: Must add to `@JsonSubTypes` above for it to work in configuration.

**Review by udoprog — CHANGES_REQUESTED** [review]

**Comment by nezdolik** [comment]
applied review comment @udoprog

File: modules/kafka/src/main/java/com/spotify/ffwd/kafka/KafkaPartitioner.java [hunk 3]
```diff
@@ -22,14 +22,16 @@
...
 import java.util.function.Supplier;
 import java.util.zip.CRC32;
 
 @JsonTypeInfo(use = JsonTypeInfo.Id.NAME, include = JsonTypeInfo.As.PROPERTY, property = "type")
 @JsonSubTypes({
     @JsonSubTypes.Type(value = KafkaPartitioner.Tag.class, name = "tag"),
     @JsonSubTypes.Type(value = KafkaPartitioner.Hashed.class, name = "static"),
-    @JsonSubTypes.Type(value = KafkaPartitioner.Host.class, name = "host")
+    @JsonSubTypes.Type(value = KafkaPartitioner.Host.class, name = "host"),
+    @JsonSubTypes.Type(value = KafkaPartitioner.RandInt.class, name = "randint")
```
Comment (udoprog) [inline modules/kafka/src/main/java/com/spotify/ffwd/kafka/KafkaPartitioner.java]: name should probably be just `random`. `int` is just implementation detail.

**Review by udoprog — APPROVED** [review]

**Pull request merged**